tests/baselines/reference/arrayFrom.types

```diff
@@ -31,7 +31,7 @@ const inputALike: ArrayLike<A> = { length: 0 };
 const inputARand = getEither(inputA, inputALike);
 >inputARand : ArrayLike<A> | Iterable<A>
 >getEither(inputA, inputALike) : ArrayLike<A> | Iterable<A>
->getEither : <T>(in1: Iterable<T>, in2: ArrayLike<T>) => ArrayLike<T> | Iterable<T>
+>getEither : <T>(in1: Iterable<T>, in2: ArrayLike<T>) => Iterable<T> | ArrayLike<T>

```

Comment:
I didn't expect to see baseline changes. Why did this one change?

Reply:
Hmm, I'm not sure but I just concluded that the change is benign. This is prone to how `.id`s are assigned to types, right?

Reply:
You're right, it's probably due to the earlier call to getReturnTypeOfSignature.